### DISCUSSION TITLE
SUGERENCIAS DE PAGINAS -  BIBLIOTECA DE RECURSOS

### DISCUSSION THREAD

**Discussion by VanillaNilla** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#218] [Q&A]
Hola hola compañeros en esta aventurilla...Aquí andamos algo atrasados pero con toda la actitud. 
El día de hoy vengo con una duda rápida. ¿Qué paginas recomiendan para encontrar recursos  gráficos? Por ejemplo tipo freepik, pixabay y así. (Me interesa mas que nada para imágenes/ilustraciones dentro de mis paginas y documentación).

Tenía mi lista de sitios favoritos para esto, pero ciertamente la perdí hace tiempo. Y me gustaría volver a tener bibliotecas de recursos. 

Muchas gracias!! Cualquier cosa que se necesite igual estamos a sus ordenes <3

**Comment by doguedogue** [comment]
Exacto, como dices hay muchísimas 🚀

https://coverr.co/
https://pixabay.com/images/search/pics/
https://unsplash.com/s/photos/free-pic

**Comment by JLuisPrz** [comment]
Hola, checa esta [página](https://rbgescuela.com/tutoblogs/10-mejores-webs-recursos-graficos-disenar/), tiene recomendaciones de webs en donde puedes encontrar muchos recursos.
